test/workbox-cli/node/app.js

```diff
@@ -71,6 +71,8 @@ describe(`[workbox-cli] app.js`, function() {
           expect(
             loggerErrorStub.alwaysCalledWithExactly(errors['invalid-common-js-module'])
           ).to.be.true;
+          /* eslint-disable */
+          console.log(error.message);
```

Comment:
Did you intend to keep this?

Reply:
No but I'm trying to figure out the AppVeyor issue (I stopped yesterday after realising this was going to be painful)